src/docs/background/10-custom-pricing/index.md

```diff
@@ -0,0 +1,263 @@
+        lineItems: [
+          {
+            code: 'line-item/room-for-two',
+            unitPrice: new Money(8000, 'USD'),
+            quantity: 3,
+          },
+          {
+            code: 'line-item/baby-crib',
+            unitPrice: new Money(500, 'USD'),
+            quantity: 3,
+          },
+          {
+            code: 'line-item/discount',
+            unitPrice: new Money(25500, 'USD'),
+            percentage: -15,
+          },
+        ],
+        cardToken: 'tok_mastercard',
+      },
+    },
+    { expand: true }
+  )
+  .then(response => {
+    // respose.data contains the response data
+  });
+```
+
+With this query `response.data` looks as follows:
+
+```json
+{
+  "data": {
+    "id": {
+      "uuid": "5c8f8dfe-2d54-48c7-b533-4da71196911a"
+    },
+    "type": "transaction",
+    "attributes": {
+      "processName": "preauth-with-custom-pricing",
+      "transitions": [
+        {
+          "transition": "transition/request",
+          "createdAt": "2019-03-18T12:24:30.239Z",
+          "by": "customer"
+        }
+      ],
+      "payoutTotal": {
+        "amount": 19507,
+        "currency": "USD"
+      },
+      "processVersion": 6,
+      "createdAt": "2019-03-18T12:24:30.223Z",
+      "lastTransitionedAt": "2019-03-18T12:24:30.239Z",
+      "protectedData": {},
+      "lineItems": [
+        {
+          "code": "line-item/room-for-two",
+          "quantity": "3",
+          "reversal": false,
+          "unitPrice": {
+            "amount": 8000,
+            "currency": "USD"
+          },
+          "lineTotal": {
+            "amount": 24000,
+            "currency": "USD"
+          },
+          "includeFor": ["customer", "provider"]
+        },
+        {
+          "code": "line-item/baby-crib",
+          "quantity": "3",
+          "reversal": false,
+          "unitPrice": {
+            "amount": 500,
+            "currency": "USD"
+          },
+          "lineTotal": {
+            "amount": 1500,
+            "currency": "USD"
+          },
+          "includeFor": ["customer", "provider"]
+        },
+        {
+          "code": "line-item/discount",
+          "percentage": "-15",
+          "reversal": false,
+          "unitPrice": {
+            "amount": 25500,
+            "currency": "USD"
+          },
+          "lineTotal": {
+            "amount": -3825,
+            "currency": "USD"
+          },
+          "includeFor": ["customer", "provider"]
+        },
+        {
+          "code": "line-item/provider-commission",
+          "percentage": "-10",
+          "reversal": false,
+          "unitPrice": {
+            "amount": 21675,
+            "currency": "USD"
+          },
+          "lineTotal": {
+            "amount": -2168,
+            "currency": "USD"
+          },
+          "includeFor": ["provider"]
+        }
+      ],
+      "lastTransition": "transition/request",
+      "payinTotal": {
+        "amount": 21675,
+        "currency": "USD"
```

Comment:
This might be the order in which it actually comes from API and gets printed but for the sake of making the example clear I'd colocate the payinTotal and payoutTotal (above the line items).